api_docs/changelog.md

```diff
@@ -20,6 +20,30 @@ format used by the Zulip server that they are interacting with.
 
 ## Changes in Zulip 8.0
 
+**Feature level 200**
+
+* [`POST /register`](/api/register-queue),
+  [`GET /events`](/api/get-events): Added realm setting
+  `create_multiuse_invite_group` which represents the ID of
+  user group whose members can create reusable invite link to realm.
+
+* `PATCH /realm`: Added `create_multiuse_invite_group` parameter to
+  support changing the user group whose members can create
+  [reusable invitation links](/help/invite-new-users#create-a-reusable-invitation-link).
+
+* `POST /invites/multiuse`: Non-admin users can now use this endpoint to
+  create reusable invite links. Previously only admins users could create
+  reusable invite links.
+
+* `GET /invites`:  Endpoint response for non-admin users now includes both
```

Comment:
Small note, there's an extra space after the colon here. I always find it useful to look at the documentation changes in a browser to catch these little things.